### ISSUE TITLE
How to customize CaptureDate for an image?

### ISSUE THREAD

**Issue by timatlee** [saimn/sigal#445]
Is it possible to customize the CaptureDate for an image or video?  Setting it in the `imagename.md` file does not seem to have any bearing.

**Comment by saimn** [comment]
What do you mean by CaptureDate, the EXIF date of a photo or is it a specific EXIF keyword ?

**Comment by timatlee** [comment]
I suppose I want to define the CaptureDate of a video, which doesn't have an EXIF field. 

In my sigal site, photos are being ordered by capture date. This is working great for photos, but an included video is sorted by its modified timestemp (since it doesn't carry EXIF fields).  

I was wondering if there was a way I could define the CaptureDate in the video's associated Markdown file so that I could force it in order?

**Comment by saimn** [comment]
It's not possible to overwrite an EXIF field, but you could check in your template and fallback to the manually defined metadata if the EXIF field is empty/undefined.

**Comment by timatlee** [comment]
That works for the displaying the date in the album, but the video file is out of sort order.

My scenario is a Christmas album - I have a kids Christmas show on December 9th (for example), and photos from Christmas morning on the 25th. The video of the kids show is placed at the end of the album after Christmas morning.

If it's not possible, that's fine - I can probably adjust the modified time on the file before running through Sigal. I wanted to check that it wasn't a "me" thing not using the tool correctly.

**Comment by saimn** [comment]
Yep, it's not possible currently, but a PR to allow overriding some EXIF/IPTC metadata would be welcome ;)
